docs/reference/cli.md

```diff
@@ -419,24 +396,30 @@ quickwit service run searcher
 
 *Examples*
 
+*Start a Searcher*
+```bash
+quickwit service run searcher --config=./config/quickwit.yaml
+```
+
 *curl request and specify fields to search*
 ```bash
- curl -L "http://127.0.0.1:7280/api/v1/wikipedia/search/stream?query=clinton&searchField=body,title" 
+curl -L "http://127.0.0.1:7280/api/v1/wikipedia/search/stream?query=clinton&searchField=body,title" 
 ```
 
 *curl request stream fast field, HTTP1.1 chunked transfer encoding*
 ```bash
- curl -L "http://127.0.0.1:7280/api/v1/wikipedia/search/stream?query=tangkhul&searchField=body,title&fastField=number&outputFormat=csv"  
+curl -L "http://127.0.0.1:7280/api/v1/wikipedia/search/stream?query=tangkhul&searchField=body,title&fastField=number&outputFormat=csv"  
 ```
 
 *curl request stream fast field as binary, force HTTP2*
 ```bash
- curl -L --http2-prior-knowledge "http://127.0.0.1:7280/api/v1/myindex/search/stream?query=clinton&searchField=body,title&fastField=number&outputFormat=clickHouseRowBinary"  
+curl -L --http2-prior-knowledge "http://127.0.0.1:7280/api/v1/myindex/search/stream?query=clinton&searchField=body,title&fastField=number&outputFormat=clickHouseRowBinary"  
 ```
 
 ### service run indexer
 
-Starts a indexing server that consumes the sources from the `quickwit source add` command. The server can be terminated when all sources are consumed.
+Starts an indexing server that consumes the sources of index IDs passed in `--indexes` argument.
+If all sources of all indexes are exhausted, the server will stop.
```

Comment:
Last time I checked the server did just idle forever

Reply:
We need to check that :/

Reply:
Yes it's still the case

Reply:
Yes, I read the code and the expected behavior is to wait until the `IndexingServer` actor exits. And this will not happen unless there is an error happens when processing a message. I'm removing the sentence.